grpc-common/src/main/scala/com/avast/cactus/grpc/ContextKeys.scala

```diff
@@ -0,0 +1,28 @@
+package com.avast.cactus.grpc
+
+import io.grpc.Context
+import io.grpc.Context.Key
+
+import scala.collection.mutable
+import scala.reflect.ClassTag
+
+object ContextKeys {
```

Comment:
Please copy shorter version of [this paragraph](https://github.com/avast/cactus/blob/gRPC/grpc-common/README.md#providing-own-interceptors) to the comment.

Reply:
OK